src/sage/categories/coxeter_groups.py

```diff
@@ -792,11 +793,12 @@ def kazhdan_lusztig_cells(self, side='left'):
 
             Computing the two sided cells in `B_3`::
 
-                sage: W = CoxeterGroup('B3', implementation='coxeter3')           # optional - coxeter3
-                sage: b3_cells = W.kazhdan_lusztig_cells('two-sided')             # optional - coxeter3
-                sage: len(b3_cells)                                               # optional - coxeter3
+                sage: W = CoxeterGroup('B3', implementation='coxeter3')     # optional - coxeter3
+                sage: b3_cells = W.kazhdan_lusztig_cells('two-sided')       # optional - coxeter3
+                sage: len(b3_cells)                                         # optional - coxeter3
```

Comment:
What's the convention here, if not column 88?

Reply:
This is an actual optional package; they should be aligned so that the tag is visible in full in the formatted doc.
My editor macro aligns these at column 64; but I think it's too hard (and not necessary) to make it globally consistent.